### PR TITLE
Organization contact feature

### LINKED ISSUE #131: Allow organization managers to define organization contact person
At the moment, organization contact person can only be added when organization is added. There is no option to modify the contact person. 

Allow organization managers and admins to define the organization contact person.

Proposal 1: free text: name, phone number or email
Proposal 2: Select one of the organization managers as contact person. Extend the current user profile to contain full name and phone number.  Profile should also have privacy setting for contact information like phone number and email. Whatever has been entered to profile and is public is shown on the organization page.

# Definition of done
* [x] Organization manager can add a contact person for organization (either in organization settings or inline in the organization profile main view).
* [x] Organization manager can give the following information for the contact person: name, email, phone nr.
* [x] Organization contact person information is displayed on the organization profile.
* [x] Organization manager can edit and delete any of the details for the contact person.

# Wireframe
1. Inline editing option
on mouse hovering near the contact information pane, an edit button appears
contactpersonsettings

On clicking the button, the fields become editable and changed can be saved/discarded using the Save and Cancel button
contactpersonsettings2

2. If a tab view of organization is considered, then we can simply place fields for adding contact information for an organization there.
altorganizationprofilesettings

### PR DESCRIPTION
**Pull request by brylie**
Closes #131

# Changes
- Cleanup unused code
- Reorganize files
- Switch 'Organization' form to modal
  - dual purpose insert/update form
- move managerIds code to schema
- fix autoform success for insert/update handling
- add #edit-organization button on organization header
- pluralize organizations route
- update organizationForm i18n texts/tokens

### CODE REVIEW

File: organization_apis/collection/helpers.js [hunk 1]
```diff
@@ -4,11 +4,19 @@ import { OrganizationApis } from './';
...
+    // Placeholder for API documents
+    let apis;
+
+    // Get organizationApis document, which joins organizations with APIs
     const organizationApis = OrganizationApis.findOne({ organizationId: this._id });
 
-    // Return array of organization apis
-    return Apis.find({ _id: { $in: organizationApis.apiIds } }).fetch();
+    // Make sure organization has APIs
+    if (organizationApis && organizationApis.apiIds.length() > 0) {
```
Comment (jykae) [inline organization_apis/collection/helpers.js]: @brylie length is not a function

```
Exception in template helper: TypeError: organizationApis.apiIds.length is not a function
    at Document.apis (http://localhost:3000/app/app.js?hash=8be05af1dd64d95a494f25d158713fcbdd148282:42836:55)
    at Object.managedApis (http://localhost:3000/app/app.js?hash=8be05af1dd64d95a494f25d158713fcbdd148282:32684:36)
    at http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:2994:16
    at http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:1653:16
    at http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:3046:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:3687:12)
    at http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:3045:27
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?hash=dcb91068ed863bb067d30163ef7df2825655ee2f:172:18)
    at Blaze.If.HTML.DIV.class (http://localhost:3000/app/app.js?hash=8be05af1dd64d95a494f25d158713fcbdd148282:32613:24)
    at Blaze.View.<anonymous> (http://localhost:3000/packages/blaze.js?hash=891d5a10d82b5122032ced96aa7f478b9b1eeda3:2677:44)
```

**Comment by jykae** [comment]
@brylie On update, would expect to remain on organization profile page, rather than going to catalog, as it behaves now.

File: organization_apis/collection/helpers.js (hunk 1 above)
Comment (brylie) [inline organization_apis/collection/helpers.js]: What are the steps to reproduce this bug?

**Comment by brylie** [comment]
@jykae changes made.

**Comment by manzapanza** [comment]
@brylie On the new modal organization form, the contact fields has a red asterisk, but are required only if one of them is filled. Maybe the red asterisk should be visible only if one of them is filled?

**Comment by brylie** [comment]
@manzapanza changes made.

@jykae or @manzapanza please review and merge, if possible, ASAP.

**Comment by jykae** [comment]
Works, merging

**Pull request merged**